src/Resources/WhatsInInspectionReportResource.cs

```diff
@@ -1,19 +1,13 @@
 ﻿namespace Linn.Purchasing.Resources
 {
+    using System.Collections.Generic;
+
     using Linn.Common.Reporting.Resources.ReportResultResources;
 
     public class WhatsInInspectionReportResource
     {
-        public string PartNumber { get; set; }
-
-        public string Description { get; set; }
-
-        public string OurUnitOfMeasure { get; set; }
-
-        public decimal QtyInStock { get; set; }
-
-        public decimal QtyInInspection { get; set; }
+        public IEnumerable<WhatsInInspectionReportEntryResource> PartsInInspection { get; set; }
 
-        public ReportReturnResource OrdersBreakdown { get; set; }
+        public ReportReturnResource BackOrderData { get; set; }
```

Comment:
are they really back orders?

Reply:
I'm not sure really. This little ResultsModel/ReportReturnResource is just a copy of the data in this footer of the old report
![Screenshot 2022-04-21 092248](https://user-images.githubusercontent.com/16683709/164412639-a0a540bc-dcb7-422a-a0c6-b77dbd1d8df8.png)

Do you think there's a better name for it?

Reply:
no fair enough. perfect name